core/controllers/editor_test.py

```diff
@@ -1191,19 +1195,22 @@ def test_reverting_to_old_exploration(self):
         csrf_token = self.get_new_csrf_token()
 
         # May not revert to any version that's not 1.
-        for rev_version in (-1, 0, 2, 3, 4, '1', ()):
+        for rev_version in (-1, 0, 2, 3, 4, '10', ()):
             response_dict = self.post_json(
                 '/createhandler/revert/%s' % self.EXP_ID, {
                     'current_version': 2,
                     'revert_to_version': rev_version
                 }, csrf_token=csrf_token, expected_status_int=400)
 
             # Check error message.
-            if not isinstance(rev_version, int):
-                self.assertIn('Expected an integer', response_dict['error'])
-            else:
+            try:
+                rev_version = int(rev_version)
```

Comment:
Can you remove this logic and rather move the case when the version is not into in a separate test case?